.github/README_pt-br.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README.md">
+        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```

Comment:
Just a small comment. Please, change here `25px` to `20px` as it isn't English translation. And for Brazilian flag : `20px` to `25px`.

```suggestion
        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```

Reply:
My bad, i'm completely forgot to change flags size  when it is the currently language